lib/uniform/blueprint.ex

```diff
@@ -382,7 +382,14 @@ defmodule Uniform.Blueprint do
             (file :: String.t(), Uniform.App.t() -> String.t())
             | (file :: String.t() -> String.t())
         ) :: term
-  defmacro modify(path_or_regex, {:fn, _, _} = fun) do
+  defmacro modify(path_or_regex, {:&, _, [{:/, _, _}]} = fun) do
```

Comment:
Moved this clause above the following one. Previously pattern matched `{:&, _, _}`, but that matches for both `&Foo.bar/1` and `& &1 <> "foo"`.

To match only `&Foo.bar/1` we need to a more specific pattern: `{:&, _, [{:/, _, _}]}`.